src/components/Table/_table.scss

```diff
@@ -1 +1,23 @@
 @import 'TableHead/table-head';
+@import '~carbon-components/scss/components/data-table/data-table';
+
+.#{$prefix}--toast {
```

Comment:
This should be scoped to the table via a class applied in edit mode, otherwise this is a globally applied style.

Values, colors, etc should be tokens